src/squidpy/gr/_utils.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 import pandas as pd
 from anndata import AnnData
-from anndata._core.views import ArrayView, SparseCSCView, SparseCSRView
```

Comment:
Could do a conditional import based on version of installed anndata and then no need for the pin in `pyproject.toml`

Something like:

https://github.com/scverse/anndata/blob/8e9eb882ddbef3fb2043a93d6d0553813dd2bc2b/src/anndata/compat/__init__.py#L33-L42

but importing old with old name/import new+rename to old